reviews/sandbox/2022-wasmedge-annual.md

```diff
@@ -0,0 +1,155 @@
+* Enhanced host SDKs in C, Rust, and Go languages.
+* A host / Wasm data binding toolkit called the `wasmedge-bindgen`.
+* A new plugin system that makes WasmEdge native host functions and extensions easier to develop and install.
+* A lot of performance improvements.
+
+The WasmEdge community has expanded its collaboration with many open-source projects in the cloud-native space in the last year. Some of these projects now have production customers using WasmEdge.
+
+* OpenFunction (CNCF project) supports WasmEdge as a container runtime.
+* WebAssembly Languages Runtime, maintained by VMWare, supports WasmEdge as a runtime
+* LF Edge eKuiper and shifu uses WasmEdge to process data streamed from IoT devices.
+* Kuasar supports WasmEdge as a container runtime.
+* containerd's runwasi supports WasmEdge as a container runtime.
+* youki, the OCI Container Runtime, supports WasmEdge as a container runtime.
+* Essa-rs ueses WasmEdge to execute serverless functions on the essa-rs platform.
+* The OpenGauss and libsql communities used WasmEdge to power UDFs (User Defined Functions) in database queries.
+* Docker Desktop integrates WasmEdge to support Wasm containers.
+* Apache Traffic Server uses WasmEdge as a plugin to process the traffic flow.
+
+Furthermore, the WasmEdge project is used by numerous commercial companies including Docker, VMWare, FutureWei, Huawei, 5miles, and ByteDance. You can check out the full list [here](https://wasmedge.org/docs/contribute/users).
+
+## Project goals
+
+This is the WasmEdge project's second Annual review after joining the CNCF Sandbox.
+
+For the immediate future, WasmEdge will focus on the following areas.
+
+* Technical goals
+    * Support more WebAssembly proposals and specs, such as threading, stack switching, logging, and debugging. In particular, we aim to be one of the first Wasm runtimes to fully support the WebAssembly component model spec.
+    * Improve support for microservices. We will enable WasmEdge applications to access commonly used databases, messaging services, proxies, sidecars, and other services.
+    * Improve support for data streaming functions. We will work with data streaming partners to support embedded functions in WasmEdge in those frameworks.
+    * Improve support for AI inference workloads in Wasm. We will continue adding support for popular models in computer vision, voice recognition, and document AI. We will also enable inference on LLMs such as Llama.
+    * Support AI training on Wasm. We are collaborating with ByteDance and the Ray community to support WasmEdge on Ray nodes.
+    * Support the complete Node.js API and NPM module ecosystem in WasmEdge's JavaScript runtime.
+    * Improve WasmEdge support in popular application frameworks such as Dapr, Suborbital, Fermyon, essa-rs, etc.
```

Comment:
Not sure if we should categorize Fermyon as an application framework, since it is a company, better to use something like Spin (from Fermyon)